### PR TITLE
CHORE: fix plan resources limit example

### PR DESCRIPTION
**Pull request by tomer-landesman**
we had some stuff that wasn't configured correctly in this example, and it should be fixed now

### CODE REVIEW

File: resources-limit/env0.rego
Comment (sagydr) [inline resources-limit/env0.rego]: I prefer `== 0` (it's not like it can be rational or negative)

Comment (tomer-landesman) [reply resources-limit/env0.rego]: no, as discussed in slack

File: resources-limit/env0.rego
Comment (sagydr) [inline resources-limit/env0.rego:8]: @tomer-landesman last question
so say 7 resources are changed. at first we have 0 approvers, so both conditions would be true and result will be pending.
After an approval this rule will be ignored? as `count(approvers) != 0` ? or what will be the result after 1 approvals?

**Review by sagydr — APPROVED** [review]
approved with comment

File: resources-limit/env0.rego
Comment (tomer-landesman) [reply resources-limit/env0.rego:8]: after an approval, the first rule won't evaluate, because it an AND gate between the conditions. the `resources < 5` also won't be evaluated, so the only thing left is the `approvers >= 1` which will result in `allow`

**Pull request merged**
